simulation/node.go

```diff
@@ -734,3 +722,13 @@ func (node *Node) setupMode() {
 func (node *Node) onUartWrite(data []byte) {
 	_, _ = node.virtualUartPipe.Write(data)
 }
+
+func (node *Node) detectVirtualTimeUART() {
+	// Input newline to both Virtual Time UART and stdin and check where node outputs newline
```

Comment:
```suggestion
	// Input newline to both Virtual Time UART and stdin and check whether node outputs newline
```

Reply:
Hi @jwhui 
The intention was really to check **where** node outputs the newline.
If the device uses virtual time UART, it will output newline via **UDP events**. Otherwise, it will output newline via **STDOUT**.

Reply:
Ah, yes. Thanks for the clarification.